### DISCUSSION TITLE
Add ability to set status from mobile app

### DISCUSSION THREAD

**Discussion by mskelton** [community/community#47670] [Mobile]
### Select Topic Area

Product Feedback

### Body

It would be helpful to set my GitHub status from the mobile app if I'm away from my computer and forgot to update my status.  Especially helpful when on vacation or other time off where access to a computer is limited.

**Comment by Turesinndaniss** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> It would be helpful to set my GitHub status from the mobile app if I'm away from my computer and forgot to update my status.  Especially helpful when on vacation or other time off where access to a computer is limited.
